src/physics/base/Interaction.hh

```diff
@@ -43,6 +43,9 @@ struct Interaction
     // Return an interaction indicating all state changes have been applied
     static inline CELER_FUNCTION Interaction from_processed();
 
+    // Return an interaction representing the creation of a new track
+    static inline CELER_FUNCTION Interaction from_spawned();
```

Comment:
Does the newly set interaction get used anywhere? It might be time to reexamine the `Interaction` class and how it's used, especially once we get a feeling of how MSC will integrate.

Reply:
Not really.. spawned and processed are both "unchanged" actions (used in post-processing), but we don't use either anywhere else (though I have used them for debugging).